### PR TITLE
Feature/cloud 2532 mongodb users

### PR DESCRIPTION
**Pull request by RohitSaini04**
# Description

Fixes [#2532](https://usxpress.atlassian.net/browse/CLOUD-2532)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
  functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
- https://octopus.usxpress.io/app#/Spaces-2/projects/mongodb-user-test/deployments/releases/0.1.3-rohit-test-octop0006/deployments/Deployments-68059

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: kubernetes.tf [hunk 1]
```diff
@@ -1,7 +1,8 @@
 resource "kubernetes_secret" "tls_secret" {
   metadata {
-    name      = "tls-certificate-${var.name}"
+    name      = "issuer-cert-${var.name}"
```
Comment (vibindaniel) [inline kubernetes.tf:3]:
```suggestion
    name      = "mongo-${var.name}"
```

Comment (vibindaniel) [inline kubernetes.tf:3]:
```suggestion
    name      = "mongo-issuer-${var.name}"
```

File: kubernetes.tf [hunk 2]
```diff
@@ -20,6 +21,7 @@ resource "kubernetes_manifest" "issuer" {
     metadata = {
       name      = var.name
```
Comment (vibindaniel) [inline kubernetes.tf:22]:
```suggestion
      name      = "mongo-${var.name}"
```

Comment (vibindaniel) [reply kubernetes.tf:22]: Update kube 51 as well

File: kubernetes.tf [hunk 3]
```diff
@@ -20,6 +21,7 @@ resource "kubernetes_manifest" "issuer" {
...
       namespace = var.namespace
+      labels    = var.tags
```
Comment (vibindaniel) [inline kubernetes.tf:24]: Provide the labels instead of tags. Mage can send both.

File: kubernetes.tf [hunk 4]
```diff
@@ -37,10 +39,10 @@ resource "kubernetes_manifest" "certificate" {
     metadata = {
       name      = var.certificate_name
       namespace = var.namespace
-      labels    = var.certificate_labels
+      labels    = var.tags
```
Comment (vibindaniel) [inline kubernetes.tf:42]: Same here. Labels instead of Tags

File: kubernetes.tf [hunk 5]
```diff
@@ -37,10 +39,10 @@ resource "kubernetes_manifest" "certificate" {
...
     }
     spec = {
-      secretName = var.certificate_name
+      secretName = "m-u-${var.certificate_name}"
```
Comment (vibindaniel) [inline kubernetes.tf:45]: This should be modified at terraform-variant-apps instead of here.

**Review by parulpachoriusx — APPROVED** [review]

**Pull request merged**